client/src/utils.js

```diff
@@ -0,0 +1,9 @@
+import moment from 'moment';
+
+export const apiUrl = 'http://localhost:1012';
+
+export function formatTime (rawDate) {
+    const dateObject = moment(rawDate);
+    const formattedDate = dateObject.format('MMMM D, YYYY');
+    return formattedDate;
+}
```

Comment:
best practice is to leave an empty line as the very last line of the file. Consider adding an empty line after this line

Reply:
Thank you!